### DISCUSSION TITLE
Design

### DISCUSSION THREAD

**Discussion by dattavelivela** [ykdojo/defaang#144] [Q&A]
Hi Folks. I'm Datta, An Aspiring UX Designer. 

I'm here to contribute to this project as a Designer and I would love to get started contributing to open source as a designer (UI, UX)

Let me know if there is anything that I can help with design related?

Thank you for your time.

**Comment by ykdojo** [comment]
Thank you @dattavelivela!

What do you think about improving the UI of the submission form somehow?

You should be able to find it here (you need to sign up / log in first): https://defaang.vercel.app/submit
